tools/packaging/kernel/build-kernel.sh

```diff
@@ -97,6 +99,7 @@ Options:
 	-b <type>    	: Enable optional config type.
 	-c <path>   	: Path to config file to build the kernel.
 	-d          	: Enable bash debug.
+	-D              : Enable device mapper support.
```

Comment:
Related to the above. Is this enabling devicemapper, or dm-verity? Can you be as explicit as possible in the description here.

Reply:
Done.